lib/gui/index.ts

```diff
@@ -27,6 +29,8 @@ export interface ServerArgs {
 }
 
 export default (args: ServerArgs): void => {
+    overrideTestplaneConfig(args.toolAdapter.toolName, args.toolAdapter.config as TestplaneConfigAdapter);
```

Comment:
I'm not sure that we should override user config in gui mode. Because to my mind user must set the required values himself. I would just change the default value of option.

Reply:
Decided to do it using a toggle in Settings panel in UI.